src/melodyPlayer/Keyboard.jsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable functional/immutable-data */
+import * as React from 'react';
+import styles from './keyboard.module.css';
+
+export function Keyboard() {
+  const totalKeyss = [0, 1, 2, 3];
+  const keyNames = [
+    'C',
+    'C#/Db',
+    'D',
+    'D#/Eb',
+    'E',
+    'F',
+    'F#/Gb',
+    'G',
+    'G#/Ab',
+    'A',
+    'A#/Bb',
+    'B',
+  ];
+  const keyboardDiv = document.createElement('div');
+  keyboardDiv.setAttribute('id', 'keyboard');
+  keyboardDiv.classList.add(styles.keyboard);
```

Comment:
looks like this style isn't getting added because `keyboard` isn't a class in my css file?  How do I apply this style?

Reply:
looks like changing keyboard in the style sheet to a class made it work
<img width="577" alt="Screen Shot 2022-04-20 at 4 52 51 PM" src="https://user-images.githubusercontent.com/29072865/164321043-e1a79bd6-5ab2-4e34-8e58-0234476d6402.png">

But now I'm running into a couple new issues:

1. The key divs are no longer bordered
2. The old resize event listeners no longer fire - do I need to add to/make a new hook for these?